include/SDL3/SDL_notification.h

```diff
@@ -0,0 +1,116 @@
+/*
+  Simple DirectMedia Layer
+  Copyright (C) 1997-2023 Sam Lantinga <slouken@libsdl.org>
+
+  This software is provided 'as-is', without any express or implied
+  warranty.  In no event will the authors be held liable for any damages
+  arising from the use of this software.
+
+  Permission is granted to anyone to use this software for any purpose,
+  including commercial applications, and to alter it and redistribute it
+  freely, subject to the following restrictions:
+
+  1. The origin of this software must not be misrepresented; you must not
+     claim that you wrote the original software. If you use this software
+     in a product, an acknowledgment in the product documentation would be
+     appreciated but is not required.
+  2. Altered source versions must be plainly marked as such, and must not be
+     misrepresented as being the original software.
+  3. This notice may not be removed or altered from any source distribution.
+*/
+
+/**
+ *  \file SDL_notification.h
+ *
+ *  \brief Header file for notification API
+ */
+
+#ifndef SDL_notification_h_
+#define SDL_notification_h_
+
+#include <SDL3/SDL_stdinc.h>
+
+#include <SDL3/SDL_begin_code.h>
+/* Set up for C function definitions, even when using C++ */
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * \brief SDL_Notification flags.
+ */
+typedef enum SDL_NotificationFlags
+{
+    SDL_NOTIFICATION_PRIORITY_LOW        = 0x00000010,   /**< lowest */
+    SDL_NOTIFICATION_PRIORITY_NORMAL     = 0x00000020,   /**< normal/medium */
+    SDL_NOTIFICATION_PRIORITY_HIGH       = 0x00000040    /**< high/important/critical */
+} SDL_NotificationFlags;
+
+/**
+ * \brief SDL_Icon flags.
+ */
+typedef enum SDL_IconFlags
+{
+    SDL_ICON_TYPE_SINGLE_FILE    = 0x00000010,   /**< A single icon file. */
```

Comment:
Are these to be used as bit flags? Looking at their use in the strict, they cannot be used together so I'd use the default enum values.

If you intend to use bit flags, an enum does not support `|` in c++ (without operator function overrides), so you'd need to use macros.